frontend/src/pages/profile-preferences-page/common/profile-preferences-page-form.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC, UpdateProfileValue } from 'common/types/types';
+import style from '../styles.module.scss';
+import React from 'react';
+import { useAppForm } from 'hooks/hooks';
+import { profileUpdateValidationSchema } from '../../../validation-schemas/validation-schemas';
+import { Input, Loader } from '../../../components/common/common';
+
+type Props = {
+  onSubmit: { (submitValue: UpdateProfileValue): Promise<void> };
+  isLoading: boolean;
+  defaultFormValue: { firstName: string; lastName: string; username: string };
+  error: string | undefined;
+};
+
+const ProfilePreferencesPageForm: FC<Props> = ({ onSubmit, defaultFormValue, error, isLoading }) => {
+  const { control, errors, handleSubmit } = useAppForm<UpdateProfileValue>({
+    defaultValues: defaultFormValue,
+    validationSchema: profileUpdateValidationSchema,
+  });
+
+  return (
+    <form className={style['profile-settings-block']} onSubmit={handleSubmit(onSubmit)}>
+      <div className={style['profile-setting-change-block']}>
+        <div className={style['profile-setting-input-block']}>
+          <Input
+            control={control}
+            errors={errors}
+            name="username"
+            type="text"
+            inputClassName={style['input']}
+            label="username"
+            placeholder="enter new user name"
+          />
+          <p>Update your username</p>
```

Comment:
```suggestion
            label="Username"
            placeholder="Enter new username..."
          />
```
Paragraphs under every field look redundant.

Reply:
done